components/06-examples/samenwerken.config.yml

```diff
@@ -0,0 +1,53 @@
+notes: |
+  De aside met de checklist schuift mee met het formulier, tot hij een punt bereikt waar hij niet verder MAG scrollen.
+  We gebruiken hiervoor Bootstrap Affix, en deze heeft twee values - `data-offset-top` en `data-offset-bottom` - die bepalen waar het kantelpunt zit.
+  Let op de uitlijning die te zien is in onderstaande instructie afbeelding.
+
+  Zie ook
+  * [Bootstrap Affix](https://getbootstrap.com/docs/3.3/javascript/#affix)
+  * Positioneren van de affix, links in het menu onder het kopje "Documentation"
```

Comment:
Ook hier, deze notes gaan over het progress block component wat hier niet gebruikt wordt

Reply:
Deze is uitgevoerd. Zo niet corrigeer me waar nodig.